src/common/internal/test_group.ts

```diff
@@ -0,0 +1,411 @@
+import { Fixture, SkipTestCase, TestParams, UnexpectedPassError } from '../framework/fixture.js';
```

Comment:
This file has not changed (except for the imports); it just didn't get detected as a rename for some reason.